react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleStart = this.handleStart.bind(this);
+    this.handleStop = this.handleStop.bind(this);
+    this.handleReset = this.handleReset.bind(this);
+    this.state = { start: false };
+    this.interval = null;
+    this.seconds = 0;
+  }
+
+  handleStart() {
+    const timeDisplay = document.querySelector('.time');
+    this.interval = setInterval(() => {
+      timeDisplay.textContent = this.seconds;
+      this.seconds++;
+    }, 1000);
+    this.setState({ start: true });
+  }
+
+  handleStop() {
+    clearInterval(this.interval);
+    this.setState({ start: false });
+  }
+
+  handleReset() {
+    this.handleStop();
+    this.seconds = 0;
+    const timeDisplay = document.querySelector('.time');
+    timeDisplay.textContent = this.seconds;
+  }
+
+  render() {
+    const start = this.state.start;
+    let symbol;
+    const circleElement = document.querySelector('.circle');
+    if (start) {
+      symbol = <Stop onClick={this.handleStop} />;
+      circleElement.removeEventListener('click', this.handleReset);
+    } else {
+      symbol = <Start onClick={this.handleStart} />;
+      circleElement.addEventListener('click', this.handleReset);
+    }
+
+    return (
+      <div>
+        {symbol}
+      </div>
+    );
+  }
```

Comment:
Use state to display the time "seconds" and to determine showing play or pause icon